scripts/run-integration-tests-gcloud.sh

```diff
@@ -80,7 +78,7 @@ function fund_ip() {
 
   wait_until_node_is_ready "${ip}"
   eth_address=$(get_eth_address "${ip}")
-  fund_if_empty "${eth_address}" "${provider}" "${hopr_token_contract}"
+  fund_if_empty "${eth_address}" "${provider}"
```

Comment:
Isn't passing `provider` deprecated?

Reply:
Yes, but I didn't want to fix this script in its entirety just yet.